tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -5,6 +5,22 @@
     wait_for_connection:
       delay: 5
       timeout: 180
+  - name: Expose engine VM webui over a local port via ssh port forwarding
+    command: >-
+      sshpass -e ssh -o ServerAliveInterval=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -g -L
+      {{ he_webui_forward_port }}:{{ he_fqdn }}:443 {{ he_fqdn }}
+    environment:
+      - "{{ he_cmd_lang }}"
+      - SSHPASS: "{{ he_appliance_password }}"
+    changed_when: true
+    async: 14400
+    poll: 0
+  - name: Open the relevant port on firewalld
```

Comment:
Why open the port **after** the ssh tunneling ? 
I think you should open the port first.

Reply:
Done